src/mono/wasm/Wasm.Build.Tests/Blazor/BuildPublishTests.cs

```diff
@@ -45,7 +45,10 @@ public void DefaultTemplate_WithoutWorkload(string config)
     [InlineData("Release")]
     public void DefaultTemplate_NoAOT_WithWorkload(string config)
     {
-        string id = $"blz_no_aot_{config}_{Path.GetRandomFileName()}_{s_unicodeChar}";
+        // disable relinking tests for Unicode: github.com/emscripten-core/emscripten/issues/17817
```

Comment:
Open an issue in the runtime repo to track this, and use that in a commented out `// [ActiveIssue("..")]`, so it can be found if we grep for it.